tools/templates/vfvs_run.py

```diff
@@ -267,19 +400,175 @@ def process_ligand(task):
+            with open(docking_file, 'w') as f: 
+                f.writelines(['Receptor'])
+                f.writelines([config_['receptor'] + '\n'])
+                f.writelines(['RMSD'])
+                f.writelines([config_['rmsd'] + '\n'])
+                f.writelines(['Binding pocket'])
+                f.writelines(['{} {}'.format( config_['min_x'], config_['max_x']) ])
+                f.writelines(['{} {}'.format( config_['min_y'], config_['max_y']) ])
+                f.writelines(['{} {}\n'.format( config_['min_z'], config_['max_z']) ])
+                f.writelines(['Number of binding poses'])
+                f.writelines([config_['n_poses'] + '\n'])
+                f.writelines(['Ligands list'])
+                f.writelines([ligand_list_file + '\n'])
+                f.writelines(['END'])
+                
+            with open(ligand_list_file, 'w') as f: 
+                f.writelines(task['ligand_path'])
+                
+        if(task['program'] == 'gold'):
+            
+            with open(task['config_path']) as fd:
+                config_ = dict(read_config_line(line) for line in fd)
+            for item in config_: 
+                if '#' in config_[item]: config_[item] = config_[item].split('#')[0]
+            
+            conf_file = cmd[-1]
+            
+            with open(conf_file, 'a+') as f: 
+                f.writelines(['  GOLD CONFIGURATION FILE\n'])        
+                f.writelines(['  AUTOMATIC SETTINGS'])        
+                f.writelines(['autoscale = 1\n'])        
+                f.writelines(['  POPULATION']) 
+                f.writelines(['popsiz = auto'])        
+                f.writelines(['select_pressure = auto'])        
+                f.writelines(['n_islands = auto'])        
+                f.writelines(['maxops = auto'])        
+                f.writelines(['niche_siz = auto\n'])        
+                f.writelines(['  GENETIC OPERATORS'])        
+                f.writelines(['pt_crosswt = auto'])        
+                f.writelines(['allele_mutatewt = auto'])        
+                f.writelines(['migratewt = auto\n'])        
+                f.writelines(['  FLOOD FILL'])        
+                f.writelines(['radius = {}'.format(config_['radius'])])        
+                f.writelines(['origin = {}   {}   {}'.format(config_['center_x'], config_['center_y'], config_['center_z'])])
+                f.writelines(['do_cavity = 0'])        
+                f.writelines(['floodfill_center = point\n'])        
+                f.writelines(['   DATA FILES'])        
+                f.writelines(['ligand_data_file {} 10'.format(task['ligand_path'])])        
+                f.writelines(['param_file = DEFAULT'])        
+                f.writelines(['set_ligand_atom_types = 1'])        
+                f.writelines(['set_protein_atom_types = 0'])        
+                f.writelines(['directory = {}'.format(conf_file.split('.')[0])])        
```

Comment:
Is this a directory that needs to be created prior to gold running? or will it create it?

Reply:
This is a bug that needs to be updated (will do so).

Reply:
Just double-checked: this is a directory created during the docking run :)